config/schema/ocha_ai_chat.schema.yml

```diff
@@ -162,18 +162,6 @@ ocha_ai_chat.plugin.embedding:
       type: integer
       label: 'Max tokens.'
 
-# Source plugin base settings.
-ocha_ai_chat.plugin.source:
-  type: mapping
-  label: 'Source plugin base settings.'
-  mapping:
-
-# Text extractor plugin base settings.
-ocha_ai_chat.plugin.text_splitter:
```

Comment:
There is a typo there, it should be `ocha_ai_chat.plugin.text_extractor`.

That aside, why the removal of the "source", "text_extractor" and "vector_store" base plugin settings?